.github/workflows/test_dbt_models.yaml

```diff
@@ -32,8 +32,10 @@ jobs:
         run: echo "TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%S")" >> "$GITHUB_ENV"
         shell: bash
 
+      # Only triggered when called by a bot. Otherwise, notifications
+      # go to whoever called the workflow
       - name: Send failure notification
-        if: failure()
+        if: github.event_name == 'workflow_dispatch' && github.triggering_actor == 'sqoop-bot[bot]' && failure()
```

Comment:
There's really no reason for this SNS topic to be used if the workflow is called by a human, since they'll get a GH notification through the normal channels. Thus, I've gated it to only notify us when triggered by `sqoop-bot` will test once this is merged by manually triggering a `sqoop-bot` rebuild.